package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This is only for tests now? WDYT about putting it alongside tests so it's clear it's only for tests, and also to prepare for testing with multiple versions...